.taplo.toml

```diff
@@ -0,0 +1,30 @@
+include = ["Cargo.toml", "**/*.toml"]
+
+[formatting]
+# Align consecutive entries vertically.
+align_entries = true
```

Comment:
I'm not sure if I like this, as the reformating in order to preserve this property when adding new entries affects a lot of lines, making git diffs larger (so harder to see what changed)

Reply:
Personally, I think it's an OK trade-off for the readability benefits

Reply:
I think the giant diff is a bigger issue than having some part of the files aligned, I just used a default config and reverted it not to apply aligning.